src/components/files/FileBrowser.vue

```diff
@@ -0,0 +1,104 @@
+<template>
+	<template v-if="!items.length">
+		<NoFiles />
+	</template>
+	<template v-else>
+		<div class="flex flex-wrap gap-3">
+			<Folder
+				v-for="folder in folders"
+				:modelValue="folder"
+				@update:modelValue="updateItem(folder)"
+			/>
+		</div>
+		<div class="mt-3 flex flex-wrap gap-3">
+			<File v-for="file in files" :modelValue="file" @update:modelValue="updateItem(file)" />
```

Comment:
Is this not equivalent to the following?
```suggestion
			<File v-for="file in files" :key="file.id" v-model="file" />
```